internal/backend/runtime/omni/controllers/omni/talos_extensions.go

```diff
@@ -30,7 +29,7 @@ import (
 type TalosExtensionsController = qtransform.QController[*omni.TalosVersion, *omni.TalosExtensions]
 
 // NewTalosExtensionsController instantiates the TalosExtensions controller.
-func NewTalosExtensionsController() *TalosExtensionsController {
+func NewTalosExtensionsController(imageFactoryClient *imagefactory.Client) *TalosExtensionsController {
```

Comment:
Switch to using the new wrapper, so the only point where we create a new image factory GRPC client is actually in `magefactory.Client`